admin/class-social-facebook.php

```diff
@@ -1,211 +0,0 @@
-				break;
-			case 'already_exists':
-				$return = "<p class='notice-error notice'><span>" . __( 'This Facebook user has already been added as an admin.', 'wordpress-seo' ) . '</span></p>';
-				break;
-			default:
-				$return = '';
-				break;
-		}
-
-		return $return;
-	}
-
-	/**
-	 * This method will hook into the defined get params
-	 */
-	private function get_listener() {
-		$delfbadmin = filter_input( INPUT_GET, 'delfbadmin' );
-		if ( ! empty( $delfbadmin ) ) {
-			$this->delete_admin( $delfbadmin );
-		}
-		elseif ( filter_input( INPUT_GET, 'fbclearall' ) ) {
-			$this->clear_all();
-		}
-	}
-
-	/**
-	 * Deletes the admin from the options
-	 *
-	 * @param string $delfbadmin Facebook admin ID.
-	 */
-	private function delete_admin( $delfbadmin ) {
-		$this->verify_nonce( 'delfbadmin' );
-
-		$admin_id = sanitize_text_field( $delfbadmin );
-		$option   = WPSEO_Options::get( 'fb_admins' );
-
-		if ( isset( $option[ $admin_id ] ) ) {
-			$fbadmin = $option[ $admin_id ]['name'];
-			unset( $option[ $admin_id ][ $admin_id ] );
-			WPSEO_Options::set( 'fb_admins', $option );
-
-			/* translators: %s expands to the username of the removed Facebook admin. */
-			$this->success_notice( sprintf( __( 'Successfully removed admin %s', 'wordpress-seo' ), $fbadmin ) );
-
-			unset( $fbadmin );
-		}
-
-		unset( $admin_id );
-
-		// Clean up the referrer url for later use.
-		if ( ! empty( $_SERVER['REQUEST_URI'] ) ) {
-			$this->cleanup_referrer_url( 'nonce', 'delfbadmin' );
-		}
-	}
-
-	/**
-	 * Clear all the facebook that has been set already
-	 */
-	private function clear_all() {
-		$this->verify_nonce( 'fbclearall' );
-
-		// Reset to defaults, don't unset as otherwise the old values will be retained.
-		WPSEO_Options::set( 'fb_admins', WPSEO_Options::get_default( 'wpseo_social', 'fb_admins' ) );
-
-		$this->success_notice( __( 'Successfully cleared all Facebook Data', 'wordpress-seo' ) );
-
-		// Clean up the referrer url for later use.
-		if ( ! empty( $_SERVER['REQUEST_URI'] ) ) {
-			$this->cleanup_referrer_url( 'nonce', 'fbclearall' );
-		}
-	}
-
-	/**
-	 * Clean up the request_uri. The given params are the params that will be removed from the URL
-	 */
-	private function cleanup_referrer_url() {
-		$_SERVER['REQUEST_URI'] = remove_query_arg(
-			func_get_args(),
-			sanitize_text_field( $_SERVER['REQUEST_URI'] )
-		);
-	}
-
-	/**
-	 * When something is going well, show a success notice
-	 *
-	 * @param string $notice_text Message string.
-	 */
-	private function success_notice( $notice_text ) {
-		add_settings_error( 'yoast_wpseo_social_options', 'success', $notice_text, 'updated' );
-	}
-
-	/**
-	 * Verify the nonce from the URL with the saved nonce
-	 *
-	 * @param string $nonce_name Nonce name string.
-	 */
-	private function verify_nonce( $nonce_name ) {
-		if ( wp_verify_nonce( filter_input( INPUT_GET, 'nonce' ), $nonce_name ) !== 1 ) {
-			die( "I don't think that's really nice of you!." );
-		}
-	}
-}
```

Comment:
This file is still loaded in vendor/composer/autoload_classmap.php & vendor/composer/autoload_static.php

Reply:
A composer du fixed this.